recipes/plotapi/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "plotapi" %}
+{% set version = "6.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 352761ecff3c7cd36c15180af01df50f991c2a223e19b89dbb72306575dde7a0
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.7.1,<4.0.0
```

Upstream does not declare a min and max Python. Conda-forge's min is 3.6.
```suggestion
    - python >=3.6
```